pkg/config/config.go

```diff
@@ -105,6 +110,13 @@ func LoadArgs(args ...string) (ProgArgs, error) {
 	pArgs.RTE.TimeUnitToLimitEvents = time.Second
 
 	flags.BoolVar(&pArgs.Version, "version", false, "Output version and exit")
+	flags.StringVar(&pArgs.DumpConfig, "dump-config", "",
```

Comment:
In case the user hasn't specified a file path, we are not providing a default dump-config file path and choosing to not dump the config. This is also applicable if dump-config is explicitly specified to be an empty string. Might be worth mentioning this in the flag description so that the user does not assume that a default file path would be picked up and the config would still be dumped.

Reply:
that's a good point. Will change in the next upload.